_posts/en/posts/2024-07-03-disclose_already_asked_for.md

```diff
@@ -45,7 +45,7 @@ Responsibly disclosed by John Newbery, claiming discovery by Amiti Uttarwar and
 - 2020-09-21 Pieter Wuille opens [PR #19988](https://github.com/bitcoin/bitcoin/pull/19988) as a comprehensive approach to fixing this and other bugs
 - 2020-10-14 Pieter's PR is merged
 - 2021-01-14 Bitcoin Core version 0.21.0 is released with a fix
-- 2022-04-25 The last vulnerable Bitcoin Core version (0.20.0) goes EOL
+- 2022-04-25 The last vulnerable Bitcoin Core version (0.20.x) goes EOL
```

Comment:
The title states that versions up to 0.20.2 were vulnerable, but the timeline refers to 0.20.0 as the last vulnerable version.